src/unistd.rs

```diff
@@ -3355,6 +3355,55 @@ mod getres {
     }
 }
 
+feature! {
+#![feature = "process"]
+#[cfg(target_os = "freebsd")]
+pub mod rfork_freebsd {
+    use crate::errno::Errno;
+    use crate::Result;
+    use super::ForkResult;
+    use super::Pid;
+
+    libc_bitflags! {
+        /// Flags for [`rfork`]
+        /// subset of flags supported by FreeBSD 12.x and onwards
```

Comment:
```suggestion
        /// Flags for [`rfork`]
        ///
        /// subset of flags supported by FreeBSD 12.x and onwards
```

We should add a newline here or the they will be rendered into a single paragraph